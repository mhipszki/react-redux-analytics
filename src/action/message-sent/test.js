import messageSent from './';

test('generates action to indicate start of message sending', () => {
  Date.now = jest.fn(() => 1234567890);
  expect(messageSent()).toMatchSnapshot();
});
