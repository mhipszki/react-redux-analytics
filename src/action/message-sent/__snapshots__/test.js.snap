// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates action to indicate start of message sending 1`] = `
Object {
  "timestamp": 1234567890,
  "type": "MESSAGE_SENT",
}
`;
